### DISCUSSION TITLE
Issue with Honda HJet HA–420

### DISCUSSION THREAD

**Discussion by GrimCultZero** [maartentamboer/X-Touch-Mini-FS2020#128] [Configurations]
Hi everyone,
Can somebody advise please, when I'm loading into Honda HJet HA–420 all the knobs and buttons are working fine, except the Attitude (encoder 3 in my case). Here is the log window, but I have a hard time understanding what and where I need to fix here...
![image](https://user-images.githubusercontent.com/126724797/226414174-c0419975-f6c4-464e-9056-8bdada703063.png)

**Comment by maartentamboer** [comment]
So this can be 2 things
1. Do you have the mobiflight plugin installed?
2. It could be that your airplane does not have support for certain simvars, this causes  the "crashes" in the templates to resolve this you can remove that functionality from the config file.

You can remove this from the default config by removing the following lines:
https://github.com/maartentamboer/X-Touch-Mini-FS2020/blob/main/Configurations/config_default.json#L148-L152
https://github.com/maartentamboer/X-Touch-Mini-FS2020/blob/main/Configurations/config_default.json#L178-L182

I would be better to add your own config file for this specific jet. You can take a look at the `Configurations` directory to see how this is done with the A320, CJ4 and CRJ. See also the documentation https://dev-tty.nl/X-Touch-Mini-FS2020/configuration/global/

**Reply by GrimCultZero** [reply]
Thank you so much @maartentamboer! I will try all this. Thank you for your work!
